tests/main/prepare-image-grub/task.yaml

```diff
@@ -1,12 +1,15 @@
 summary: Check that prepare-image works for grub-systems
 
-systems: [-ubuntu-core-*, -fedora-*, -opensuse-*, -arch-*, -amazon-*, -centos-*]
+# ubuntu-14.04: systemd-run not supported
+# ubuntu-core-*: building images is not explicitly supported
+# fedora-*: building images is not explicitly supported
+# centos-*: building images is not explicitly supported
+# arch-*: building images is not explicitly supported
+# amazon-*: building images is not explicitly supported
+systems: [-ubuntu-core-*, -fedora-*, -opensuse-*, -arch-*, -amazon-*, -centos-*, -ubuntu-14.04*]
```

Comment:
since there's so many systems we don't run this test on, perhaps it would be simpler to do an explicit list of what we do support, i.e.
```suggestion
# building images is only supported on classic ubuntu, but ubuntu 14.04 does not have systemd-run to perform the test
systems: [ubuntu-16*, ubuntu-18*, ubuntu-2*]
```